src/Shape/Box.js

```diff
@@ -71,66 +70,99 @@ export default class Box extends BaseClass {
     }
 
     const outlierData = [];
+    let orient = "vertical";
```

Comment:
You shouldn't be storing this as a global variable outside of each group of data points (it's possible that a user could want certain boxes to be vertical and certain boxes to be horizontal).